### PR TITLE
kubeadm: add note about a bug in the PublicKeysECDSA feature gate

### PR DESCRIPTION
**Pull request by SequoiaGod**
Add note about that in the FG section of the kubeadm init doc.
xref https://github.com/kubernetes/website/issues/46799

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 1]
```diff
@@ -171,7 +171,8 @@ Feature gate descriptions:
 `PublicKeysECDSA`
 : Can be used to create a cluster that uses ECDSA certificates instead of the default RSA algorithm.
 Renewal of existing ECDSA certificates is also supported using `kubeadm certs renew`, but you cannot
-switch between the RSA and ECDSA algorithms on the fly or during upgrades.
+switch between the RSA and ECDSA algorithms on the fly or during upgrades. Kubernetes {{< skew currentVersion >}} has
```
Comment (Shubham82) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md:174]:
```suggestion
switch between the RSA and ECDSA algorithms on the fly or during upgrades. Kubernetes {{< skew currentVersion >}} has
```

Comment (Shubham82) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md:174]: Small nit: remove trailing whitespace.

**Comment by Shubham82** [comment]
The preview looks good to me.
https://deploy-preview-46878--kubernetes-io-main-staging.netlify.app/docs/reference/setup-tools/kubeadm/kubeadm-init/#feature-gates

/lgtm

**Review by neolit123 — COMMENTED** [review]
/lgtm
/sig cluster-lifecycle
/triage accepted
/assign @sftim 

there seems to be a netlify failure here.

**Comment by Shubham82** [comment]
@SequoiaGod The test cases failed due to Netlify build timeout failures.
please re-push the commit.

```
git reset
git commit --amend --no-edit
git push --force-with-lease
```

**Comment by sftim** [comment]
/remove-label tide/merge-method-squash

not needed; keeping a note of the merge base is handy

/approve

**Pull request merged**